packages/flutter/lib/widgets.dart

```diff
@@ -35,6 +35,7 @@ export 'src/widgets/dismissible.dart';
 export 'src/widgets/disposable_build_context.dart';
 export 'src/widgets/drag_target.dart';
 export 'src/widgets/draggable_scrollable_sheet.dart';
+export 'src/widgets/dual_transition_builder.dart';
```

Comment:
If we export it here it'll be public for use.

Reply:
Confirmed, there're conflicts in animations package if we export it.

![image](https://user-images.githubusercontent.com/15884415/83935217-2305f400-a7ea-11ea-9a5f-350bfdc55bca.png)